drv/user-leds/src/main.rs

```diff
@@ -35,7 +35,11 @@ use userlib::*;
 
 cfg_if::cfg_if! {
     // Target boards with 4 leds
-    if #[cfg(any(target_board = "gemini-bu-1", target_board = "gimletlet-2"))] {
+    if #[cfg(any(
+            target_board = "gemini-bu-1",
+            target_board = "gimletlet-1",
```

Comment:
Out of curiosity, I see several deltas for supporting gimletlet-1; do you actually have one you're using?

Reply:
OH YEAH!
![IMG_5982](https://user-images.githubusercontent.com/745333/152379430-a4e85457-c437-4955-95be-821dbfe06605.JPG)